### ISSUE TITLE
[Bug]: NotSupportedError: Failed to load because no supported source was found.

### ISSUE THREAD

**Issue by hix1000** [FreeTubeApp/FreeTube#5597]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

fails to fetch video

### Expected Behavior

plays as normal

### Issue Labels

accessibility issue, content not loading, feature stopped working, usability issue

### FreeTube Version

v0.21.3 Beta

### Operating System Version

Windows 11

### Installation Method

.exe

### Primary API used

Invidious API

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by absidue** [comment]
Please try a different Invidious instance, Invidious users are getting the same problem on Invidious itself so it's definitely an Invidious problem and not something wrong on FreeTube's side.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Thanks for documenting this @hix1000 ! I will have to close this as we cant do anything about this but if others run into this we can point them to this issue.
